docs/tutorials/python/types_of_data_augmentation.md

```diff
@@ -302,7 +302,7 @@ Some shortcut functions are provided to perform multiple augmentation in a singl
 ```python
 # A random crop, with a random resizing, and random aspect ratio jitter
 example_image_copy = example_image.copy()
-aug = mx.image.RandomSizedCropAug(size=(100, 100), min_area=0.1, ratio=(1.0, 1.5))
+aug = mx.image.RandomSizedCropAug(size=(100, 100), area=0.1, ratio=(1.0, 1.5))
```

Comment:
So looks like people are already using min_area as keyword argument.
We should keep backward compatibility by adding **kwargs to RandomSizedCropAug and random_size_crop and check if min_area is present, then issue a warning and override area

Reply:
Do we have a standard warning format for deprecated parameters?

Reply:
no. somehting like xx is deprecated and renamed to xxx